src/plugins/settings/base-setting/base-setting.ts

```diff
@@ -130,8 +118,8 @@ export abstract class UIPSetting extends UIPPlugin {
    * Disable setting
    * By default is used when there are no setting's targets
    */
-  public set disabled(force: boolean) {
-    this.$$attr('disabled', force);
+  public setDisabled(force: boolean): void {
+    return;
```

Comment:
Is it problem for us to have no disable attribute on the element itself ?

Reply:
@ala-n actually no, lets keep it